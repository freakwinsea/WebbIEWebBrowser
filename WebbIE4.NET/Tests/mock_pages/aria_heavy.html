<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>ARIA Heavy Test</title>
</head>
<body>
    <h1>Accessibility Test</h1>

    <div role="button" tabindex="0" aria-label="Close dialog">X</div>

    <nav aria-label="Main Navigation">
        <a href="/dashboard" role="link" aria-label="Go to user dashboard">My Dashboard</a>
    </nav>

    <form>
        <label for="search-input">Search</label>
        <input type="search" id="search-input" aria-label="Search the site">
        <button type="submit" aria-label="Submit search">Go</button>
    </form>

    <div aria-hidden="true">
        <p>This content should be ignored by the parser.</p>
        <a href="/hidden-link">Hidden Link</a>
    </div>

</body>
</html>
