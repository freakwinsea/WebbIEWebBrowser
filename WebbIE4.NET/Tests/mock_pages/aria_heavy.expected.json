[
  {
    "id": "webbie-id-0",
    "type": "h1",
    "text": "Accessibility Test",
    "href": ""
  },
  {
    "id": "webbie-id-1",
    "type": "button",
    "text": "Close dialog",
    "href": ""
  },
  {
    "id": "webbie-id-2",
    "type": "link",
    "text": "Go to user dashboard",
    "href": "/dashboard"
  },
  {
    "id": "webbie-id-3",
    "type": "text",
    "text": "Search",
    "href": ""
  },
  {
    "id": "webbie-id-4",
    "type": "search",
    "text": "Search the site",
    "href": ""
  },
  {
    "id": "webbie-id-5",
    "type": "submit",
    "text": "Submit search",
    "href": ""
  }
]
